meet_balls_app/views.py

```diff
@@ -15,6 +16,16 @@ def home(request):
     return render(request, 'meet_balls_app/home.html')
 
 
+def notifications(request):
+    notifications = Notification.objects.filter(player=request.user.player)
```

Comment:
Could notifications can be None (there aren't notifications to this user)?
If so, do you think there should be a special thing to do on this scenario?

Reply:
Added. Resolving conversation.